src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,48 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class StringTest {
+    @DisplayName("콤마를 기준으로 값이 분리된다.")
+    @Test
+    void split_value_with_comma() {
+        String given = "1,2";
+
+        String[] result = given.split(",");
+
+        assertThat(result).containsExactly("1", "2");
+    }
+
+    @DisplayName("콤마가 없을 때에도 콤마를 기준으로 값이 분리된다.")
+    @Test
+    void split_valueWithoutComma_with_comma() {
+        String given = "1";
+
+        String[] result = given.split(",");
```

Comment:
```suggestion
                    String[] result = "1".split(",");
```

로컬 변수 없이 구현해도 되지 않을까?